src/nxdoc/web-ui/web-ui-release-notes/web-ui-11-1-release-notes.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Nuxeo Web UI 11.1 Release Notes
+description: Discover changes brought in our recent Nuxeo Web UI updates.
+review:
+    comment: ''
+    date: '2020-06-24'
+    status: ok
+toc: true
+labels:
+tree_item_index: 999
+---
+
+{{! multiexcerpt name='release-notes'}}
+
+{{! multiexcerpt name='matching-notes'}}
+{{#> callout type='info' heading='Upgrade Notes'}}
+This page mentions what's new. Refer to the [upgrade notes]({{page page='web-ui-11-1-upgrade-notes'}}) to transition to this version.
+{{/callout}}
+{{! /multiexcerpt}}
+
+## What's New in Web UI 11.1
+
+### Upload Experience for Large Files
+
+Web UI provides a safe experience when uploading files, even for large files in the 75gb range and when the upload goes on for hours due to a slow connection.
+
+If a network problem happens, upload is resumed whenever possible. Otherwise, guidance is provided so that you can decide what to do with the situation: e.g. create the document anyway and upload the attachment later.
```

Comment:
The part about resume is not exactly true. We don't have this on the default handler, and with S3, the client only guarantees that it will retry a few times before giving up, so even if the connection comes back up, it won't resume the upload.

Reply:
Thanks! Removing that part to prevent confusion.